index.js

```diff
@@ -1,6 +1,28 @@
 const level = require('level');
 const db = connectToDatabase('./leveldb');
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
+
+(async function() {
+    await acceptStudent('201811827', 'Norhani A. Ayaon', 22, 'Marawi City');
+}());
 
 function connectToDatabase(db){
     return level(db, { valueEncoding: 'json' });
 }
+
+async function acceptStudent(id, fullName, age, address){
+    const student = { 
+        ID: id, 
+        Name: fullName, 
+        Age: age, 
+        Address: address,
+    };
+    await db.put(id, student);
+    db.get(id, function(err, value){
+        if(err){
+            console.log(err);
+        } else{
+            console.log(value, 'Status: ', status[0]);
+        }
+    })
```

Comment:
store status in student object. ... 
```suggestion
    const student = { 
        ID: id, 
        Name: fullName, 
        Age: age, 
        Address: address,
        Status : status[0]
    };
    await db.put(id, student);
    console.log(value, 'Status: ', student.Status);
```